src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -28,56 +28,84 @@
       :on-press            #(rf/dispatch [:navigate-to :wallet-address-watch])
       :add-divider?        true}]]])
 
-(def account-cards
-  [{:name                "Account 1"
-    :balance             "€0.00"
-    :percentage-value    "€0.00"
-    :customization-color :blue
-    :type                :empty
-    :emoji               "🍑"
-    :on-press            #(rf/dispatch [:navigate-to :wallet-accounts])}
-   {:customization-color :blue
-    :on-press            #(rf/dispatch
-                           [:show-bottom-sheet {:content new-account}])
-    :type                :add-account}])
+(defn- add-account-placeholder
+  [color]
+  {:customization-color color
+   :on-press            #(rf/dispatch [:show-bottom-sheet {:content new-account}])
+   :type                :add-account})
 
 (def tabs-data
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
+(defn get-balance
+  [balances address]
+  (.toFixed (->> balances
+                 (filter #(= (:address %) address))
+                 first
+                 :balance)
+            2))
+
+(defn account-cards
+  [{:keys [accounts loading? balances profile]}]
+  (let [accounts-with-balances (mapv (fn [account]
+                                       (assoc account
+                                              :type                :empty
+                                              :customization-color (:customization-color profile)
+                                              :on-press            #(rf/dispatch [:navigate-to
+                                                                                  :wallet-accounts])
+                                              :loading?            loading?
+                                              :balance             (str "$"
+                                                                        (get-balance balances
+                                                                                     (:address
+                                                                                      account)))))
+                                     accounts)]
+    (conj accounts-with-balances (add-account-placeholder (:customization-color profile)))))
+
+(defn- view-internal
+  []
+  (let [accounts     (rf/sub [:profile/wallet-accounts])
+        top          (safe-area/get-top)
+        selected-tab (reagent/atom (:id (first tabs-data)))
+        loading?     (rf/sub [:wallet-2/tokens-loading?])
+        balances     (rf/sub [:wallet-2/balances])
+        profile      (rf/sub [:profile/profile])]
+    [rn/view
+     {:style {:margin-top top
```

Comment:
`:options   {:insets {:top? true` should be used in the screens namespace instead

Reply:
@mmilad75 `:options   {:insets {:top? true` works fine